docs/upstream-differences.md

```diff
@@ -40,22 +57,24 @@ See https://github.com/openshift/installer/compare/release-4.10...jewzaam:releas
 
 # Introducing new OCP release into ARO RP
 
-To support a new version of OpenShift on ARO, you will need to reconcile [upstream changes](https://github.com/openshift/installer) with our [forked installer](https://github.com/jewzaam/installer-aro). This will not be a merge, but a cherry-pick of patches we've implemented.
+To support a new version of OpenShift on ARO, you will need to reconcile [upstream changes](https://github.com/openshift/installer) with our [forked installer](https://github.com/openshift/installer-aro). This will not be a merge, but a cherry-pick of patches we've implemented.
 
 ## Update installer fork
 
 To bring new OCP release branch into ARO installer fork:
 
-1. Assess and document differences in X.Y and X.Y-1 in upstream
+1. If not done already, fetch our fork and upstream repos:
     ```sh
     # clone our forked installer
-    git clone https://github.com/jewzaam/installer-aro.git
+    git clone https://github.com/openshift/installer-aro.git
```

Comment:
I think we may need to fork `openshift/installer-aro` to your account first and then clone it (or you can clone `openshift/installer-aro` and add remote `<account>/installer` afterward instead) because now we can't simply push to `openshift/installer-aro` and need PR to reflect the change.